### ISSUE TITLE
Question on using Detail23K for analyzing EOS decisions

### ISSUE THREAD

**Issue by yilun-hua** [yuezih/less-is-more#3]
Thank you for the interesting work! The paper says that the analysis of EOS decisions (Section 2) uses Detail23K, which is part of the model's training data. I wonder if the same analysis can be done with datasets the models were not trained on. Are there some advantages/specific reasons to use Detail23K?

**Comment by yuezih** [comment]
Thanks for your interest! That's really a good question.

1. Using Detail23K (which is part of the training data) is **not intentional**. In fact, we were not aware of this issue, so thank you for bringing it to mind. 
2. By manipulation the multimodal context (e.g. adding noise to the image, concatenating another image to the image, and masking initial words of the text), the perturbed context actually **differs from** the training data. So IMHO, our final conclusions should therefore be similarly derived using other data that models were not trained on.

**Comment by yilun-hua** [comment]
Thank you for your reply! This is very helpful!
